CameraOverlay/overlay_all_stats.py

```diff
@@ -35,7 +36,6 @@ def on_connect(self, client, userdata, flags, rc):
 		self.base_canvas.draw_text("Power:", (5, self.text_height * 2))
 		self.base_canvas.draw_text("Cadence:", (5, self.text_height * 3))
 		self.base_canvas.draw_text("Distance:", (5, self.text_height * 4))
-		self.base_canvas.draw_text("Message:", (5, self.text_height * 5 ), size = 1)
```

Comment:
Repeating what I said on monash-human-power/dashboard#67, why is this moved off the base overlay?

Reply:
I was thinking having the word "Message" on the overlay while messages are not being sent is not useful and will be distracting for the rider, and for the message to pop up only when its actually being sent.  I am alright with having it back on too.

Reply:
That makes complete sense, keep it like this :) my bad